### PR TITLE
Improve warning when BSQ swap offer is removed

### PR DESCRIPTION
**Pull request by ripcurlx**
Instead of this:
Bildschirmfoto 2021-12-03 um 12 12 27
Do this:
Bildschirmfoto 2021-12-03 um 12 19 50
Bildschirmfoto 2021-12-03 um 12 20 02

### CODE REVIEW

File: desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersViewModel.java [hunk 1]
```diff
@@ -181,6 +183,11 @@ boolean isBootstrappedOrShowPopup() {
         return GUIUtil.isBootstrappedOrShowPopup(p2PService);
     }
 
+    public boolean isMakerFee(OpenOffer openOffer) {
+        Coin makerFee = openOffer.getOffer().getMakerFee();
+        return makerFee != null && makerFee.isPositive();
```
Comment (ghost) [inline desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersViewModel.java]: `makerFee != null` check seems to be redundant

File: desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersViewModel.java [hunk 2]
```diff
@@ -181,6 +183,11 @@ boolean isBootstrappedOrShowPopup() {
         return GUIUtil.isBootstrappedOrShowPopup(p2PService);
     }
 
+    public boolean isMakerFee(OpenOffer openOffer) {
```
Comment (chimp1984) [inline desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersViewModel.java]: Maybe better `hasMakerFee`

**Review by chimp1984 — DISMISSED** [review]
utACK

File: desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersViewModel.java (hunk 1 above)
Comment (ripcurlx) [reply desktop/src/main/java/bisq/desktop/main/portfolio/openoffer/OpenOffersViewModel.java]: Correct - I moved some code parts and didn't revisit if the null check is still necessary.

**Review by bisq-github-admin-3 — APPROVED** [review]
utACK - based on https://github.com/bisq-network/bisq/pull/5891#pullrequestreview-823198496

**Pull request merged**